lib/phoenix/socket/message.ex

```diff
@@ -30,7 +30,7 @@ defmodule Phoenix.Socket.Message do
       }
     rescue
       err in [KeyError] ->
-        raise Phoenix.Socket.InvalidMessageError, message: "missing key #{inspect err.key}"
+        reraise Phoenix.Socket.InvalidMessageError, message: "missing key #{inspect err.key}"
```

Comment:
This is wrong, you are not passing the stacktrace (and the previous code is fine). This is a false positive from credo.